salt/top.sls

```diff
@@ -25,6 +25,7 @@ base:
 
   'covid19-dev':
     - covid19
+    - covid19public
```

Comment:
Might be cleaner to structure in directory structure, e.g.

```
covid19:
init.sls public.sls
```

Reply:
make changes as you specified